lte/gateway/deploy/roles/magma/tasks/main.yml

```diff
@@ -341,12 +340,9 @@
 - name: Install gmock and gtest for C++ testing
   become: yes
   ignore_errors: yes
-  shell: cmake CMakeLists.txt && make && cp *.a /usr/lib
+  shell: cmake . && cmake --build . --target install
   args:
-    chdir: /usr/src/{{ item }}
-  with_items:
-    - gtest
-    - gmock
+    chdir: /usr/src/googletest
```

Comment:
Any idea why this worked before?

Reply:
See my answer to the `Install multipath-tools` above.